integration-tests/deployment/keystone/changeset/0001_deploy_registry.go

```diff
@@ -0,0 +1,22 @@
+package changeset
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	kslib "github.com/smartcontractkit/chainlink/integration-tests/deployment/keystone"
+)
+
+func Apply0001(lggr logger.Logger, env deployment.Environment, ab deployment.AddressBook, registryChainSel uint64) (deployment.ChangesetOutput, error) {
```

Comment:
note - we found that leaving these unordered is actually easier. Since you will have multiple higher environments (staging/testnet/mainnet) where you may want to run them in different orders/skip some etc. Can take a look at the CCIP dir for naming there.